openedx/adg/lms/webinars/validators.py

```diff
@@ -0,0 +1,30 @@
+"""
+Validators for webinars
+"""
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.utils.translation import gettext_lazy as _
+
+from .helpers import extract_emails_from_string
+
+
+def validate_comma_separated_emails(emails_string):
+    """
+    Validate a string comma seperated emails by checking if all the emails are valid.
+    Raise a validation error if any of the emails is invalid.
+
+    Args:
+        emails_string (str): String containing the comma separated emails
+    """
+    try:
+        emails = extract_emails_from_string(emails_string)
+        already_seen = {}
+        for email in emails:
+            if email in already_seen:
+                raise ValidationError(_('Please do not enter duplicate emails!'))
+
+            already_seen[email] = True
+            validate_email(email)
+
+    except Exception:
```

Comment:
Can we be specific in the Exception type?

Reply:
Actually this is intentional, I want to catch any kind of error that might arise here.